### DISCUSSION TITLE
How to change font type for one page?

### DISCUSSION THREAD

**Discussion by guo-jia-qi** [daattali/beautiful-jekyll#1125] [Q&A]
How to change font type of one page (not main page)?

**Comment by daattali** [comment]
You'll need to know some basic CSS. Add a CSS file, and use the YAML frontmatter parameters to add this css file to this page.

**Reply by guo-jia-qi** [reply]
The default font is proportional, right? I'd like to change it to monospace font in order to justify text. Besides changing font, is there any other ways to realise justified alignment?
